app/mailers/qualification/expiration_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class Qualification::ExpirationMailer < ApplicationMailer
+  REMINDER_TOMORROW = 'qualification_expiration_reminder_tomorrow'
+  REMINDER_1_YEAR = 'qualification_expiration_reminder_1_year'
+  REMINDER_2_YEARS = 'qualification_expiration_reminder_2_years'
+
+  def reminder(person, content_key)
```

Comment:
```suggestion
  def reminder(person, reminder_for)
```

`content_key = "qualification_expiration_reminder_#{reminder_for}"`